app/serializers/articles/index_serializer.rb

```diff
@@ -0,0 +1,10 @@
+class Articles::IndexSerializer < ActiveModel::Serializer
+  include ActionView::Helpers::TextHelper
+
+  attributes :id, :title, :body
+
+  def body
+    truncate(object.body, length: 75)
+  end
+
+end
```

Comment:
I can't see any specs for this serializer

Reply:
Specs for Index serializer added.